### ISSUE TITLE
Color inputs are not consistent with the color code that generated the display

### ISSUE THREAD

**Issue by PicarWang** [edelstone/material-palette-generator#10]
For example, when I input "d21f3e", the color value displayed in the palette is "d21f3d"

**Comment by maxloh** [comment]
This repo is a reverse engineered clone of [Google's Material Design color tool](https://material.io/design/color/the-color-system.html#tools-for-picking-colors).

The behavior you mentioned also exists in Google's own tool so I don't think it is a bug.

**Comment by edelstone** [comment]
Hi @PicarWang,

That is odd behavior. I'm not able to reproduce that with the color you gave (see screenshot below).

Can you tell me more about your browser and operating system?

<img width="835" alt="Screen Shot 2021-05-19 at 9 36 16 AM" src="https://user-images.githubusercontent.com/8269046/118831930-c6c6a380-b885-11eb-9d44-eced31fd109d.png">

**Comment by edelstone** [comment]
Interesting, I see that now. This is Google's official color system and I'm really just borrowing their giant Javascript file in order to make this site, so I'm hesitant to try to make changes to that. But I will leave this issue open in case others want to comment.

**Comment by vkottler** [comment]
Yeah @PicarWang that's the same behavior if you use the Google-hosted app, I've been trying to pick apart the algorithm(s) in general to understand what they're doing. Thanks to this project I grabbed the source and un-minified it, hoping to get some Python that can generate color palettes in the "Material Design" style (talking to myself here: https://github.com/vkottler/svgen/issues/6).
